extensions/amp-ad/0.1/amp-ad-3p-impl.js

```diff
@@ -150,6 +195,7 @@ export class AmpAd3PImpl extends AMP.BaseElement {
    * @override
    */
   onLayoutMeasure() {
+    this.lifecycleReporter_.sendPing('adRequestStart');
```

Comment:
I guess I was thinking of `adRequestStart` as start of the entire process, not just the actual network request, which is what `sendAdRequest` is for.  Do you have a suggestion for what to call this one, other than `adRequestStart`?  Recall that we really do want to compare counts of `onLayoutMeasure` invocations between the two branches.